### DISCUSSION TITLE
Feature Request: Add Restart Server in settings because Restart Frigate does not successfully restart application, frigate webui videos remain unresponsive

### DISCUSSION THREAD

**Discussion by chrislawso** [blakeblackshear/frigate#13869] [Ideas]
I would like to request feature is it possible to add a setting menu button to restart the entire host or container that frigate is running inside if not using the frigate recommended docker compose? 

One of my frigate environments are running the tteck install direct in lxc, no docker involved, and because the system is experiencing constant error problems (see https://github.com/blakeblackshear/frigate/discussions/13868 ) and over the course of sometimes hours or sometimes days the frigate system will stop recording and the live view and birdseye view will stop showing video streams. Portions of the frigate webui will continue to function and if I go to Restart Frigate this does not appear to completely reboot the lxc, and the system never restarts back to a working state. I am required to remote connect to the proxmox host and manually reboot the lxc container. 

I would like frigate webui to be able to reboot the lxc container it is running in.

Thanks

**Comment by NickM-27** [comment]
Frigate is officially supported runing in a docker container, there is no possibility for it to restart the host. 

The restart frigate button already sends SIGKILL when it's not running in it's normal environment, there is nothing more that it can do to restart

https://github.com/blakeblackshear/frigate/blob/dev/frigate%2Futil%2Fservices.py#L36

Running in the tteck lxc it shouldn't be any different but in any case that's not an officially supported way to install frigate.
